docs/farming-&-staking/staking/operators/tips-operator.mdx

```diff
@@ -213,29 +213,30 @@ services:
       - "0.0.0.0:40333:40333/tcp"
     restart: unless-stopped
     command: [
+      "run",
       "--chain", "gemini-3g",
       "--base-path", "/var/subspace",
       "--blocks-pruning", "256",
       "--state-pruning", "archive-canonical",
-      "--port", "30333",
+      "--listen-on", "30333",
       "--dsn-listen-on", "/ip4/0.0.0.0/udp/30433/quic-v1",
       "--dsn-listen-on", "/ip4/0.0.0.0/tcp/30433",
       "--rpc-cors", "all",
       "--rpc-methods", "unsafe",
-      "--rpc-external",
+      "--rpc-listen-on",
```

Comment:
Here is and in many other places syntax is wrong, this is not a boolean option

Reply:
Sorry, what we should replace `--rpc-external` with in this case? I thought it could be used with or without specifying the port, as it has a default value unless specified explicitly.

Reply:
Definitely not. `--rpc-listen-on` replaces a bunch of options, it is not a simple renaming. `--help` explains the syntax of each option.

`--rpc-listen-on` is not a port, it is `IP:port`. If you want to listen on all IPs then you specify `0.0.0.0` as IP explicitly. As to port - you specify the port you want, probably `9944`, so the value is `0.0.0.0:9944` here. In future we'll make RPC optional (Substrate currently always starts it unconditionally) and will not run at all unless this option is specified.